### ISSUE TITLE
Vaughn Live returns invalid links

### ISSUE THREAD

**Issue by markosjal** [streamlink/streamlink#1780]
### Checklist

- [x] This is a bug report.
- [ ] This is a feature request.
- [ ] This is a plugin (improvement) request.
- [X] I have read the contribution guidelines.

### Description

...

### Expected / Actual behavior

...

### Reproduction steps / Explicit stream URLs to test

1. streamlink https://vaughnlive.tv/everything80s --stream-url --best >>my.m3u
2. streamlink https://vaughnlive.tv/franky_eyes2 --stream-url --best >>my.m3u

the above result in unplayable streams:

output from file:

rtmp://192.240.105.171:1935/live? live=1 pageUrl=https://vaughnlive.tv/everything80s playpath=live_everything80s

rtmp://192.240.105.171:1935/live? live=1 pageUrl=https://vaughnlive.tv/franky_eyes2 playpath=live_franky_eyes2

### Logs
sorry no log , and no idea where to get it.

### Comments, screenshots, etc.

...

[Love Streamlink? Please consider supporting our collective. Thanks!](https://opencollective.com/streamlink/donate)

**Comment by beardypig** [comment]
Vaughnlive changed the port number for their rtmp servers the other week, I think this was fixed in 0.13.0. Make sure you are using the latest version of streamlink.

**Comment by techmouse** [comment]
I've been having problems with it too, even when using port 2935.

```
[console][error] Try 1/1: Could not open stream <RTMPStream({'pageUrl': 'https://vaughnlive.tv/wiriamu', 'flv': '-', 'live': True, 'playpath': 'live_wiriamu', 'rtmp': 'rtmp://192.240.105.171:2935/live?'}, redirect=False> (Failed to read data from stream: Read timeout)
error: Could not open stream <RTMPStream({'pageUrl': 'https://vaughnlive.tv/wiriamu', 'flv': '-', 'live': True, 'playpath': 'live_wiriamu', 'rtmp': 'rtmp://192.240.105.171:2935/live?'}, redirect=False>, tried 1 times, exiting
[console][info] Closing currently open stream...
```

**Comment by yoya3312** [comment]
The first 3 bytes of the given stream should be 'FLV' instead of 'MVN'.

**Comment by fadster** [comment]
Up until last week, I was able to play the streams directly in vlc by opening the URL passed to flv.js, namely: `https://mp4.vaughnsoft.net/live?app=live&port=2935&stream=live_<channel_name>` (also specifying the appropriate referrer). Something was then changed on the site and this method now fails with codec errors as playback is attempted. I'm assuming this is due to the modification of the first 3 bytes of the stream. Can anyone confirm this, just to satisfy my curiosity?

Thank you for your awesome work and dedication!
